packages/button/src/button.tsx

```diff
@@ -180,10 +180,11 @@ type ButtonContentProps = Pick<
 
 function ButtonContent(props: ButtonContentProps) {
   const { leftIcon, rightIcon, children, iconSpacing } = props
+  const isString = typeof children === "string"
   return (
     <>
       {leftIcon && <ButtonIcon marginEnd={iconSpacing}>{leftIcon}</ButtonIcon>}
-      {children}
+      {isString ? <chakra.span>{children}</chakra.span> : children}
```

Comment:
Kindly use our helper assertion function isString and isNumber from our utils package to determine if the span container is necessary.

Reply:
@TimKolberger 
Has been updated, I didn't add `isNumber` because pure number text will not be translated.